services/core-api/app/api/mines/permits/permit/resources/permit.py

```diff
@@ -122,8 +143,14 @@ def post(self, mine_guid):
 
         uploadedFiles = data.get('uploadedFiles', [])
 
+        permit_prefix = permit_no[0];
```

Comment:
It might be wise to add "permit prefix" as a hybrid property to the permit model, that way, in case the logic changes to get the permit prefix in the future, we only need to update the hybrid property.

Reply:
Also, semicolon! 😮

Reply:
discussed on a call and decided to keep as it is